source/reference/minio-mc/mc-sql.rst

```diff
@@ -228,28 +208,29 @@ Input Formats
 - ``.json``
 - ``.parquet``
 
-For ``.csv`` file types, use :mc-cmd:`mc sql --csv-input` to 
-specify the CSV data format. See :ref:`mc-sql-csv-format` for more 
-information on CSV formatting fields.
+For ``.csv`` file types, use :mc-cmd:`mc sql --csv-input` to specify the CSV data format.
+See :ref:`mc-sql-csv-format` for more information on CSV formatting fields.
+
+For ``.json`` file types, use :mc-cmd:`mc sql --json-input` to specify the JSON data format.
+
+For ``.parquet`` file types, :mc:`mc sql` automatically interprets the data format.
 
-For ``.json`` file types, use :mc-cmd:`mc sql --json-input` to specify
-the JSON data format.
+:mc:`mc sql` determines the type by the file extension of the target object.
+For example, an object named ``data.json`` is interpreted as a JSON file.
 
-For ``.parquet`` file types, :mc:`mc sql` automatically interprets the
-data format.
+You can query data of a supported type but a different extension if the object has the appropriate ``content-type``.
+For more information, see :mc-cmd:`mc cp --attr`.
```

Comment:
I wonder if we should include the explicit full attribute for content-type.

That is, the table on 207 could be a list table specifying the input format or the equivalent content type for the file.

Reply:
That would be handy, yes. So:
- `text/csv`
-  `application/json`
-  Whatever Parquet is.

Hmm [Parquet not yet](https://issues.apache.org/jira/browse/PARQUET-1889?focusedCommentId=17698080&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17698080), it seems

Reply:
Hah. I guess we can leave that cell blank for now then.